common/lib/units.js

```diff
@@ -2,7 +2,7 @@ import numeral from 'numeral';
 
 export const formatValue = (v) => {
   const number = parseFloat(v);
-  return number >= 1000 ? numeral(number).format('0,0.[000000]') : number;
+  return number >= 1000 ? numeral(number).format('0,0.[000]') : number;
```

Comment:
I think these should always use the full STX value, down to the smallest comment unit. We should also use bigint since the js floating point accuracy can cause issues.

Reply:
Sorry, this was specifically requested by Muneeb to show less precision for big numbers.

Reply:
I argue otherwise in the issue https://github.com/blockstack/blockstack-explorer/issues/182